int64 rcRoll
int64 rcPitch
int64 rcYaw
int64 rcThrottle
int64 rcAUX1
int64 rcAUX2
int64 rcAUX3
int64 rcAUX4
int64 plutoIndex
int8 commandType
int8 trim_roll
int8 trim_pitch
int8 isAutoPilotOn
